### PR TITLE
Upgrade to Angular 13

### PR DESCRIPTION
**Pull request by tpitts-CFI**
Task: Upgrade to Angular 13 (ae2ae260-3e4e-11ec-976e-2df811d36180)

* Kohese performs nominally
* Perform NPM audit to assess remaining vulnerabilities

### CODE REVIEW

File: kohese-portal/client/cache-worker/cache-worker.ts [hunk 1]
```diff
@@ -958,7 +958,6 @@ async function fetchMissingCacheInformation(missingCacheData){
       const timestamp = response.timestamp;
       timestamp.responseReceiptTime = responseReceiptTime;
       console.log('::: Response for getMissingCacheInfo');
-      // tslint:disable-next-line:forin
```
Comment (cfi-dphillips) [inline kohese-portal/client/cache-worker/cache-worker.ts:961]: Why are all of the tslint lines removed?

File: kohese-portal/tslint.json [hunk 2]
```diff
@@ -1,143 +0,0 @@
-{
```
Comment (cfi-dphillips) [inline kohese-portal/tslint.json:1]: Why was this file removed?

**Review by cfi-dphillips — CHANGES_REQUESTED** [review]
This commit seems to be stacked on several other recently merged requests.  Please update to lastest master to reduce the number of files to be reviewed.

File: kohese-portal/tslint.json (hunk 2 above)
Comment (tpitts-CFI) [reply kohese-portal/tslint.json:1]: tslint is fully deprecated in angular 13. angular.json no longer recognizes "lint" as a valid configuration option. I haven't checked for implementation, but it favors esLint now.

File: kohese-portal/client/cache-worker/cache-worker.ts (hunk 1 above)
Comment (tpitts-CFI) [reply kohese-portal/client/cache-worker/cache-worker.ts:961]: tslint is fully deprecated in angular 13. angular.json no longer recognizes "lint" as a valid configuration option. I haven't checked for implementation, but it favors esLint now.

**Comment by tpitts-CFI** [comment]
merged master into angular13 branch

**Comment by cfi-dphillips** [comment]
This PR appears to have merge conflicts...

**Review by gmckune — APPROVED** [review]

**Comment by tpitts-CFI** [comment]
fixed merge conflicts

**Review by cfi-dphillips — APPROVED** [review]

**Review by cfi-dphillips — APPROVED** [review]
